liquibase-couchbase/src/test/java/common/operators/TestClusterOperatorTest.java

```diff
@@ -95,7 +96,9 @@ public BucketSettings answer(InvocationOnMock invocation) {
 
     @Test
     void should_create_collection_primary_index() {
-        testClusterOperator.getCollectionOperator(collection).createCollectionPrimaryIndex(null);
+        testClusterOperator.getBucketOperator(TEST_BUCKET)
+                .getCollectionOperator(TEST_COLLECTION, TEST_SCOPE)
+                .createCollectionPrimaryIndex(null);
```

Comment:
In all the cases in this PR we use this method with null value as a parameter. Is it possible to add the method version without parameters (not to provide a useless null every time)?